CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2.4.0
+
+This is a minor feature release.
+
+Features:
+* add a `utf8-string->sha256` function, directly analogous to `utf8-string->sha1`
+* add a `file->sha256` function, equivalent to reading a file's contents as a
+  UTF-8 string and hashing the result. Uses an InputStream internally to avoid
+  reading the entire file into memory at once.
```

Comment:
Could you add something here about increasing the range of port numbers returned by `open-port-num` to the entire ephemeral port range of 49152 through 65535?